aloscene/utils/depth_utils.py

```diff
@@ -36,7 +43,11 @@ def coords2rtheta(K, size, distortion, projection="pinhole"):
     if projection == "pinhole":
         theta = torch.atan(r_d / focal)
     elif projection == "equidistant":
+        assert not isinstance(distortion, Sequence)
```

Comment:
The input is never an instance of Sequence. I suppose you mean : `assert isinstance(distortion, float)` ?